### PR TITLE
ECHO-1208: Fixed links to operations description

### PR DESCRIPTION
**Pull request by Ashes17b**

### CODE REVIEW

File: api-reference/echo-operations/README.md [hunk 1]
```diff
@@ -10,25 +10,25 @@
```
Comment (aliakseikazak) [inline api-reference/echo-operations/README.md:10]: Add a brief description of the "account_address_create_operation" operation.

File: api-reference/echo-operations/README.md [hunk 2]
```diff
@@ -10,25 +10,25 @@
...
-* [asset_issue_operation](asset_management.md#asset_issue_operation)
-* [asset_reserve_operation](asset_management.md#asset_reserve_operation) - Used to take an asset out of circulation, returning to the issuer
-* [asset_fund_fee_pool_operation](asset_management.md#asset_fund_fee_pool_operation)
-* [asset_publish_feed_operation](asset_management.md#asset_publish_feed_operation) - Publish price feeds for market-issued assets
+* [asset_create_operation](asset-management.md#asset_create_operation) - Create new asset
+* [asset_update_operation](asset-management.md#asset_update_operation) - Update options common to all assets
+* [asset_update_bitasset_operation](asset-management.md#asset_update_bitasset_operation) - Update options specific to BitAssets
+* [asset_update_bitasset_operation](asset-management.md#asset_update_bitasset_operation) - Update options specific to BitAssets
+* [asset_update_feed_producers_operation](asset-management.md#asset_update_feed_producers_operation) - Update the set of feed-producing accounts for a BitAsset
+* [asset_issue_operation](asset-management.md#asset_issue_operation)
```
Comment (aliakseikazak) [inline api-reference/echo-operations/README.md]: Add a brief description of the "asset_issue_operation" operation.

File: api-reference/echo-operations/README.md [hunk 3]
```diff
@@ -10,25 +10,25 @@
...
-* [asset_update_bitasset_operation](asset_management.md#asset_update_bitasset_operation) - Update options specific to BitAssets
-* [asset_update_feed_producers_operation](asset_management.md#asset_update_feed_producers_operation) - Update the set of feed-producing accounts for a BitAsset
-* [asset_issue_operation](asset_management.md#asset_issue_operation)
-* [asset_reserve_operation](asset_management.md#asset_reserve_operation) - Used to take an asset out of circulation, returning to the issuer
-* [asset_fund_fee_pool_operation](asset_management.md#asset_fund_fee_pool_operation)
-* [asset_publish_feed_operation](asset_management.md#asset_publish_feed_operation) - Publish price feeds for market-issued assets
+* [asset_create_operation](asset-management.md#asset_create_operation) - Create new asset
+* [asset_update_operation](asset-management.md#asset_update_operation) - Update options common to all assets
+* [asset_update_bitasset_operation](asset-management.md#asset_update_bitasset_operation) - Update options specific to BitAssets
+* [asset_update_bitasset_operation](asset-management.md#asset_update_bitasset_operation) - Update options specific to BitAssets
```
Comment (aliakseikazak) [inline api-reference/echo-operations/README.md]: Duplicate

File: api-reference/echo-operations/README.md [hunk 4]
```diff
@@ -91,11 +91,11 @@ transaction. This off-chain synchronization can be avoided using proposals.
```
Comment (aliakseikazak) [inline api-reference/echo-operations/README.md:86]: Add a brief description of the operations:
1. contract_whitelist_operation
2. to all "Sidechain" operations

File: api-reference/echo-operations/README.md [hunk 5]
```diff
@@ -91,11 +91,11 @@ transaction. This off-chain synchronization can be avoided using proposals.
 
 ## Asset Transfer
 
-* [transfer_operation](asset_transfer.md#transfer_operation) - Transfers an amount of one asset from one account to another
-* [override_transfer_operation](asset_transfer.md#override_transfer_operation) - Allows the issuer of an asset to transfer an asset from any account to any account if they have override_authority
-* [transfer_to_address_operation](asset_transfer.md#transfer_to_address_operation)
+* [transfer_operation](asset-transfer.md#transfer_operation) - Transfers an amount of one asset from one account to another
+* [override_transfer_operation](asset-transfer.md#override_transfer_operation) - Allows the issuer of an asset to transfer an asset from any account to any account if they have override_authority
+* [transfer_to_address_operation](asset-transfer.md#transfer_to_address_operation)
```
Comment (aliakseikazak) [inline api-reference/echo-operations/README.md]: Add a brief description of the "transfer_to_address_operation" operation.

**Review by aliakseikazak — COMMENTED** [review]
Added some comments

File: api-reference/echo-operations/README.md (hunk 1 above)
Comment (Ashes17b) [reply api-reference/echo-operations/README.md:10]: Fixed

File: api-reference/echo-operations/README.md (hunk 2 above)
Comment (Ashes17b) [reply api-reference/echo-operations/README.md]: Fixed

File: api-reference/echo-operations/README.md (hunk 3 above)
Comment (Ashes17b) [reply api-reference/echo-operations/README.md]: Fixed

File: api-reference/echo-operations/README.md (hunk 5 above)
Comment (Ashes17b) [reply api-reference/echo-operations/README.md]: Fixed

File: api-reference/echo-operations/README.md (hunk 4 above)
Comment (Ashes17b) [reply api-reference/echo-operations/README.md:86]: Fixed

**Review by aliakseikazak — APPROVED** [review]

**Pull request merged**